blog/tutorial/01-index.md

```diff
@@ -240,18 +252,17 @@ func handleMsgCreatePost(ctx sdk.Context, k keeper.Keeper, msg *types.MsgCreateP
 
 	return &sdk.Result{Events: ctx.EventManager().ABCIEvents()}, nil
 }
-
-
 ```
 
-After creating a post object with creator, ID and title, the message handler calls `k.CreatePost(ctx, post)`. “k” stands for [Keeper](https://docs.cosmos.network/master/building-modules/keeper.html), an abstraction used by the SDK that writes data to the store. Let’s define the `CreatePost` keeper function in a new `keeper/post.go` file.
+After creating a post object with creator, ID and title, the message handler calls `k.CreatePost(ctx, post)`. “k” stands for [Keeper](https://docs.cosmos.network/master/building-modules/keeper.html), an abstraction used by the SDK that writes data to the store. Define the `CreatePost` keeper function in a new `keeper/post.go` file.
```

Comment:
```suggestion
After creating a post object with creator, ID, and title, the message handler calls `k.CreatePost(ctx, post)`. “k” stands for [Keeper](https://docs.cosmos.network/master/building-modules/keeper.html), an abstraction used by the SDK that writes data to the store. Define the `CreatePost` keeper function in a new `keeper/post.go` file.
```

Reply:
we'll have other handler-related updates here since the handler is deprecated

Reply:
let's continue to move in the direction of "you" as the audience instead of the collective first person, see https://www.notion.so/allinbits/Tutorials-62b31701f5124b7989f2cdaa01350a08 and https://ddbeck.com/second-person-is-ok/ (lots of opinions on audience!)